### PR TITLE
AGENT-863: docs fix

### PR DESCRIPTION
**Pull request by andfasano**
Minor docs fix

### CODE REVIEW

File: docs/user/agent/add-node/add-nodes.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 ## Pre-requisites
 1. The `oc` tool must be available in the execution environment (the "user host").
-2. The user host has a valid network connection to the target OpenShift cluster to be expanded.
+2. Ensure `oc` is properly configured to manage the cluster to be expanded, ie by either:
```
Comment (bfournie) [inline docs/user/agent/add-node/add-nodes.md:5]:
```suggestion
2. Ensure `oc` is properly configured to manage the cluster to be expanded, i.e. by either:
```

File: docs/user/agent/add-node/add-nodes.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
...
+   - running `oc login` command with the required cluster credentials
```
Comment (bfournie) [inline docs/user/agent/add-node/add-nodes.md:6]:
```suggestion
   - running `oc login` command with the required cluster credentials, or
```

File: docs/user/agent/add-node/add-nodes.md [hunk 3]
```diff
@@ -2,7 +2,9 @@
...
+   - export the environment variable `$KUBECONFIG` pointing to the cluster `kubeconfig` file
```
Comment (bfournie) [inline docs/user/agent/add-node/add-nodes.md:7]:
```suggestion
   - running `export KUBECONFIG=<cluster `kubeconfig` file>`
```

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**